OpenRA.Game/WAngle.cs

```diff
@@ -79,6 +79,27 @@ public static WAngle Lerp(WAngle a, WAngle b, int mul, int div)
 			return new WAngle(aa + (bb - aa) * mul / div);
 		}
 
+		public static WAngle ChangeByStep(WAngle a, WAngle b, WAngle step_speed)
+		{
+			var aa = a.Angle;
+			var bb = b.Angle;
+			var step = step_speed.Angle;
+			if (aa > bb && aa - bb > 512)
+				aa -= 1024;
+
+			if (bb > aa && bb - aa > 512)
+				bb -= 1024;
```

Comment:
What is this calculation for?

Reply:
Some comments on such math are always a good thing. Please consider adding a couple of comments even if they seem trivial.

Reply:
Already told him about comment on math.  I have already pass the fix advice to him.

Reply:
Maybe even expose this magic values to a `TraitInfo` to auto-document them.

Reply:
So now we need to reconstruct the entire orientation and its related things,  maybe a little complex.  Castle wants provide solution to all VXL rotation conditions.

Reply:
of course, as well as an interface to help modder to set it in yaml. But it may take some time. Thanks for your advice again